app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ActiveRecord::Base
+  has_many  :practices 
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Trailing whitespace detected.